how2meet/services/email_service.py

```diff
@@ -0,0 +1,42 @@
+"""
+GMail service. Boilerplate to arbitrarily send email messages. Uses app password from app.how2meet@gmail.com account
+
+Note: Named "email_service" to avoid conflict with the built-in email module
+"""
+import os
+import smtplib
+from email.message import EmailMessage
+
+app_email = "app.how2meet@gmail.com"
+app_password = os.getenv("H2M_EMAIL_APP_PASSWORD")
+from_email = "events@how2meet.com"
```

Comment:
Also just a side note, we may want to consider making this also configurable by ENV var. Not a blocker for this PR though

Reply:
Yeah I had questions about what the right way to do this was--why I wanted you to review. Envvars would work. I'm just trying to think if there are other addresses we'll want to use like guests@how2meet.com or something idk. Leaving for now but will consider for later